src/category_theory/category.lean

```diff
@@ -114,7 +114,7 @@ end
 
 variables (α : Type u)
 
-instance [preorder α] : small_category α :=
+instance category_of_preorder [preorder α] : small_category α :=
```

Comment:
I think there is actually something to say for putting this instance in the `preorder` namespace. (Even if it stays in this file.) It could be called `preorder.small_category`.

Reply:
Good idea. I'll make this a separate PR, in any case, as it has nothing to do with the other things here.